src/main/java/com/labsynch/labseer/domain/RDKitSaltStructure.java

```diff
@@ -0,0 +1,14 @@
+package com.labsynch.labseer.domain;
+
+import org.springframework.roo.addon.javabean.RooJavaBean;
+import org.springframework.roo.addon.jpa.activerecord.RooJpaActiveRecord;
+import org.springframework.roo.addon.json.RooJson;
+import org.springframework.roo.addon.tostring.RooToString;
+
+@RooJavaBean
+@RooToString
+@RooJson
+@RooJpaActiveRecord(sequenceName = "RDKIT_STRUCTURE_PKSEQ", finders = { "findRDKitSaltStructuresByRegEquals"})
+public class RDKitSaltStructure extends RDKitStructure {
```

Comment:
They won't share table.  Hibernate/JPA/Roo automatically create a different table (was a weird pull request because I had a sql migration which I don't actually need).

Reply:
I added the rkit_salt_structure migration for the rdkit structure subclass for salts